### ISSUE TITLE
My suggestions for ESG

### ISSUE THREAD

**Issue by Neiwun** [captaincobbs/ESG-Mod#433]
1. I made a custom Vaulter faction, which has the population effect that increases the defensive manpower capacity on a system by 75 per pop. And the power of this faction is greatly increased if I discover the Sim Camp from a curiosity, which gives 0.1 industry per manpower capacity on system. Could you make the Sim Camp into a selectable faction trait? If not, then please make a faction trait that gives 0.1 industry per manpower capacity on system.

2. Could you make it so that only 1 major faction can become the ally to the pirates, by paying some dust per turn. And, if someone else wants to be the ally to the pirates, then they have to pay a higher amount of Dust per turn. Also, the pirate ships and infantry troops would increase in power depending on how much Dust they are gaining. Lastly, make it so that the ally to the pirates can invade a pirate lair either with no consequences or by paying a large amount of Dust; this would make it less annoying if the pirates make a lair or a nearby system that has 4 or 5 planets or some important resources.

3. In the custom faction screen, please remove all population effects that give a bonus on tiny, small, medium, large, and huge planets. They're not that interesting and they make it very inconvenient to scroll and see all of the available options. Personally, I would also remove that ones that give a bonus per minor faction assimilated and per backdoor, but those don't take up as much space so you can keep them if you want. I think that people are going to feel paralyzed if you display too many options.

Anyway, I appreciate your hard work and I will definitely continue using this mod even if none of these suggestions are ever implemented.

Take care :)

**Comment by captaincobbs** [comment]
1. I could make sim camp a selectable faction trait yes
2. Sorry to say, but most of these are impossible :( 
    I can however make pirate ship and infantry strength increase when you boost them.
3. I can do that, it'll be a huge pain but I'll get to it at some point.
4. I tried this earlier, if you go over 100 things start crashing and the Hissho get immediately rebellious on all systems

**Comment by Neiwun** [comment]
1. Awesome! After you do this and make it possible for custom factions to select the Militarist and Pacifist traits, then I can make a post on reddit about my favorite custom factions with the ESG mod.
2. If that's the only thing that's possible then maybe you should use your time for something else. It's up to you, but the main thing I care about is being able to destroy a pirate lair when you are allied with the pirates.
3. It's not important, so feel free to do it whenever you feel like it.
4. That's too bad.

As always, I appreciate the work that you guys are doing!

**Comment by captaincobbs** [comment]
1. What's up with the Militarist and Pacifist traits?
2. We're already planning on a buff to pirates, so it's nothing especially out of our way.
3. I'll add it to my list :)

**Comment by Neiwun** [comment]
I mentioned this about a week ago on the steam page for the ESG mod. I can't add the Militarist or Pacifist personalities to my custom factions. And you said: "Sorry! I had fixed this last update, but another change had overwritten it. I've just fixed it and confirmed that it works, so it will definitely work next patch." Which is fine, and I'm eagerly awaiting the next version :)

**Comment by captaincobbs** [comment]
Ahh, I thought you meant political traits and I was terrified for a second

**Comment by wakystuf** [comment]
On 4, for the Hissho, you can "store" Keii already in vanilla by using their faction-specific planetary improvmenets. You add one of those to the queue, it drops your Keii by 25, and then you do whatever actions will increase Keii, and you then can cancel the building whenever you want the 25 back.

I guess if you want to have more of that, then Cobbs maybe we can just create a new building (or even a few buildings) that cost 5, 10 or 15 Keii to build, and nothing else? That way it's even easier to store keii (and can be done in smaller quanta).

**Comment by captaincobbs** [comment]
From what I've seen, Keii storage generally isn't that much of a problem, so it's not an immediate concern I have. If anything, a quest rework is more likely.

**Comment by wakystuf** [comment]
Yeah, I agree. Just pointing out that we could pretty easily add that if someone has a reason why it's needed

**Comment by Neiwun** [comment]
Yes, I have stored my Keii like that in the past, but I wish I didn't have to micromanage my Keii like that. I don't think having more buildings in the queue, to store Keii, would help much. But it would be nice if the threshold for getting "Ecstatic" would be lowered, like in the Vodyani faction quest. Anyway, I was just mentioning this hoping it would be easy to fix, but I don't plan on using the Hissho, even if they got reworked.

**Comment by Neiwun** [comment]
One more thing: when you join an alliance, the dust requirement for an Economic victory changes and the victory screen will show how much dust your alliance has gained compared to the amount needed for an Economic victory. Could you make the victory screen display how much dust you have gained, without the amount from your alliance members? Because some times you want to be in an alliance, to benefit from certain Pacifist laws, but you can reach the Economic victory requirement by yourself, leave the alliance, and then win the game instantly. It would be nice to have this information displayed somewhere.

**Comment by captaincobbs** [comment]
I can't change what's displayed on that UI specifically, sorry haha

**Comment by Neiwun** [comment]
Could you display this information somewhere else? Like in the empire dust tooltip?

**Comment by Neiwun** [comment]
In the 1.3 version, you tied the terraforming changes of the Riftborn to the Biophobic trait, so could you do the same thing with the Cryphobic, Pyrophobic, and Petraphobic traits? For Petraphobic, you could make it so that gas planets are initially colonizable and advanced tech can allow the colonization of sterile and then fertile planets. Also, this Petraphobic faction would be able to terraform from fertile into sterile and then into gas planets, but not from gas into fertile. I feel like this could make the game more interesting, even though it would probably mess up any existing balance.

**Comment by captaincobbs** [comment]
I was actually planning to do this during 1.4.1, however, a massive AI bug is going to be receiving attention instead.

**Comment by captaincobbs** [comment]
Can be deleted
